dcm4che-tool/dcm4che-tool-stowrs/src/main/java/org/dcm4che3/tool/stowrs/StowRS.java

```diff
@@ -411,6 +413,7 @@ private static void stow(StowRS instance, Attributes metadata, List<String> file
             LOG.info("STOW successful!");
         } catch (Exception e) {
             LOG.error("Exception : " + e.getMessage());
```

Comment:
Use
`LOG.error("Exception during STOW", e);`
That will automatically print the stack trace and you don't have to do it manually.